lib/game.rb

```diff
@@ -0,0 +1,50 @@
+class Game
+  include Concerns::Winable
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Human.new("X"), player_2 = Human.new("O"), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    board.turn_count % 2 == 0 ? player_1 : player_2
+  end
+
+  def over?
+    draw? || won?
+  end
+
+  def draw?
+    board.full? && !won?
+  end
+
+  def winner
+    board.cells[won?.first] if won?

```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.